extension/src/runConfigs.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+export interface IExecutionConfig {
+    name: string;
+    projectName: string;
+    workingDirectory: string;
+    args: any[];
+    vmargs: any[];
+    env: { [key: string]: string; };
+    preLaunchTask: string;
+}
+
+export interface IExecutionConfigGroup {
+    default: string;
```

Comment:
What is this value for?

Reply:
`default` is the name of the default `IExecutionConfig` in `IExecutionConfigGroup`